app/main.py

```diff
@@ -1,6 +1,19 @@
-from typing import Callable
+from typing import Callable, Any
+from functools import wraps
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    results_cache = {}
+
+    @wraps(func)
+    def wrapper(*args) -> Any:
```

Comment:
Decorated function can have keyword arguments

Reply:
changes have been made, the code is ready to be reviewed by a mentor